archive/2025-08-16-security-engineer-agent-security-openai.md

```diff
@@ -0,0 +1,101 @@
+
+We are looking for a versatile engineer who thrives in ambiguity and can make meaningful contributions from day one. You should be prepared to ship solutions quickly while maintaining a high standard of quality and security.
+
+We’re looking for people who can drive innovative solutions that will set the industry standard for agent security. You will need to bring your expertise in securing complex systems and designing robust isolation strategies for emerging AI technologies, all while being mindful of usability. You will communicate effectively across various teams and functions, ensuring your solutions are scalable and robust while working collaboratively in an innovative environment. In this fast-paced setting, you will have the opportunity to solve complex security challenges, influence OpenAI’s security strategy, and play a pivotal role in advancing the safe and responsible deployment of agentic AI systems.
+
+This role is based in **San Francisco, CA**. We use a hybrid work model of 3 days in the office per week and offer relocation assistance to new employees.
+
+**You’ll be responsible for:**
+
+- Architecting security controls for agentic AI – design, implement, and iterate on identity, network, and runtime-level defenses (e.g., sandboxing, policy enforcement) that integrate directly with the Agent Infrastructure stack.
+
+- Building production-grade security tooling – ship code that hardens safety monitoring pipelines across agent executions at scale.
+
+- Collaborating cross-functionally – work daily with Agent Infrastructure, product, research, safety, and security teams to balance security, performance, and usability.
+
+- Influencing strategy & standards – shape the long-term Agent Security roadmap, publish best practices internally and externally, and help define industry standards for securing autonomous AI.
+
+
+**We’re looking for someone with:**
+
+- Strong software-engineering skills in Python or at least one systems language (Go, Rust, C/C++), plus a track record of shipping and operating secure, high-reliability services.
+
+- Deep expertise in modern isolation techniques – experience with container security, kernel-level hardening, and other isolation methods.
+
+- Hands-on network security experience – implementing identity-based controls, policy enforcement, and secure large-scale telemetry pipelines.
+
+- Clear, concise communication that bridges engineering, research, and leadership audiences; comfort influencing roadmaps and driving consensus.
+
+- Bias for action & ownership – you thrive in ambiguity, move quickly without sacrificing rigor, and elevate the security bar company-wide from day one.
```

Comment:
Bias to action was a key part of that blog by a guy that left OAI recently. I'll find the reference later. This seems to be an explicit value.